### PR TITLE
Add game Caesar 3

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

File: Applications/Games/Caesar III/Local/script.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+include(["Engines", "Wine", "QuickScript", "LocalInstallerScript"])
+include(["Engines", "Wine", "Engine", "Object"]);;
```
Comment (madoar) [inline Applications/Games/Caesar III/Local/script.js]: Can you please fix the `;` above?

File: Applications/Games/Caesar III/Steam/script.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Engine", "Object"]);;
```
Comment (madoar) [inline Applications/Games/Caesar III/Steam/script.js]: Same as above

File: Applications/Games/Caesar III/Local/script.js (hunk 1 above)
Comment (ImperatorS79) [inline Applications/Games/Caesar III/Local/script.js]: Done.

File: Applications/Games/Caesar III/Steam/script.js (hunk 2 above)
Comment (ImperatorS79) [inline Applications/Games/Caesar III/Steam/script.js]: Done.

**Comment by madoar** [comment]
Do you also want to add a french translation for the application descriptions?

**Comment by ImperatorS79** [comment]
I have to use i18n/update_translations.py, don't I ? So the code needs to be merged.

**Comment by madoar** [comment]
So you want to translate all 3 applications in one go?

**Comment by madoar** [comment]
Otherwise you can translate them individually in each PR

**Comment by ImperatorS79** [comment]
Ok, so I just have to add a line in the .po file each time I submit a script ?

**Comment by madoar** [comment]
You can use the instructions located at https://github.com/PlayOnLinux/Scripts/wiki/Translation inside each of your branches to add a translation for each branch/application.
The only disadvantage of this approach is, that you'll need to merge the branches later on, because some values inside the files will differ (things like times, dates and so on).
Alternatively we can merge your PRs first and you can translate them in one go, if you prefer that.

**Comment by ImperatorS79** [comment]
I prefer the second option :)

**Pull request merged**